app/ide-desktop/lib/dashboard/playwright.config.ts

```diff
@@ -16,8 +16,9 @@ export default test.defineConfig({
   repeatEach: process.env.CI ? 3 : 1,
   expect: {
     toHaveScreenshot: { threshold: 0 },
-    timeout: 30_000,
+    timeout: 120_000,
   },
+  timeout: 120_000,
```

Comment:
I don't think increasing timeout will fix our problem, actually.

When I look at `drag labels onto multiple rows`, I see many events are emitted one after another, without waiting for any possible effect (selecting rows maybe?) - are we sure the next event will take desired effect before the previous one will have it?

For example, in https://github.com/enso-org/enso/blob/1a8b82e23778f8e6a9e092b76fc964fb2000e1c6/app/gui2/e2e/componentBrowser.spec.ts#L68 we click at the background to deselect any selected node. After every click, we actually have a check if all the nodes are deselected - without it, sometimes we proceed before the deselecting took place and the test was flaky.

Reply:
you're right - nice catch, thanks!
i ran a bunch of tests when i was afk and got this a number of times:
```
Running 4 tests using 1 worker

  ✓  1 labels.spec.ts:5:6 › drag labels onto single row (5.5s)
  ✘  2 labels.spec.ts:37:6 › drag labels onto multiple rows (2.0m)
  ✓  3 labels.spec.ts:73:6 › drag (recursive) (7.2s)
  ✓  4 labels.spec.ts:175:6 › drag (inverted, recursive) (4.0s)


  1) labels.spec.ts:37:6 › drag labels onto multiple rows ──────────────────────────────────────────

    Test timeout of 120000ms exceeded.

    Error: expect(locator).toBeVisible()

    Locator: getByTestId('drive-view').getByRole('table').locator('tbody').getByRole('row').first().getByTestId('asset-label').getByText('aaaa')
    Expected: visible
    Received: hidden
    Call log:
      - expect.toBeVisible with timeout 120000ms
      - waiting for getByTestId('drive-view').getByRole('table').locator('tbody').getByRole('row').first().getByTestId('asset-label').getByText('aaaa')


      66 |   const assetRows = actions.locateAssetRows(page)
      67 |   const labels = actions.locateLabelsPanelLabels(page)
    > 68 |   const label = 'bbbb'
         |                                                     ^
      69 |   api.addLabel('aaaa', backend.COLORS[0])
      70 |   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
      71 |   api.addLabel(label, backend.COLORS[1]!)

        at /home/me/git/enso/app/ide-desktop/lib/dashboard/e2e/labels.spec.ts:68:83

  1 failed
    labels.spec.ts:37:6 › drag labels onto multiple rows ───────────────────────────────────────────
  3 passed (2.3m)
```

i suppose i'll have to figure out how to properly wait then

Reply:
nvm. it's not an issue with timeouts, it's an actual issue with the `onDrop` handler